core/git-utils/index.js

```diff
@@ -134,7 +134,9 @@ function describeTag(ref, opts) {
 }
 
 function diffSinceIn(committish, location, opts) {
-  const formattedLocation = slash(path.relative(opts.cwd, location));
```

Comment:
I think I'd rather just leave off the `-- <formattedLocation>` args if `formattedLocation` is empty. The git command works the same way in that case.

```diff
diff --git a/core/git-utils/index.js b/core/git-utils/index.js
index bb11df0f..720ae11a 100644
--- a/core/git-utils/index.js
+++ b/core/git-utils/index.js
@@ -134,15 +134,17 @@ function describeTag(ref, opts) {
 }
 
 function diffSinceIn(committish, location, opts) {
+  const args = ["diff", "--name-only", committish];
   const formattedLocation = slash(path.relative(opts.cwd, location));
 
+  if (formattedLocation) {
+    // avoid same-directory path.relative() === ""
+    args.push("--", formattedLocation);
+  }
+
   log.silly("diffSinceIn", committish, formattedLocation);
 
-  const diff = ChildProcessUtilities.execSync(
-    "git",
-    ["diff", "--name-only", committish, "--", formattedLocation],
-    opts
-  );
+  const diff = ChildProcessUtilities.execSync("git", args, opts);
   log.silly("diff", diff);
 
   return diff;
```